diff -up ./src/libscotch/parser.c.orig ./src/libscotch/parser.c
--- a/src/libscotch/parser.c	2018-06-18 01:41:24.677521316 +0300
+++ b/src/libscotch/parser.c	2018-06-18 01:44:12.975183090 +0300
@@ -111,13 +111,13 @@ const char * const          string)
   }
 #endif /* SCOTCH_DEBUG_PARSER1 */
 
-#ifdef COMMON_PTHREAD_MEMORY
+#ifdef SCOTCH_PTHREAD
   if (muteflag != 0) {                            /* Unsafe code with respect to race conditions but should work; portable TSL needed */
     muteflag = 0;
     pthread_mutex_init (&mutelocdat, NULL);       /* Initialize local mutex */
   }
   pthread_mutex_lock (&mutelocdat);               /* Lock local mutex */
-#endif /* COMMON_PTHREAD_MEMORY */
+#endif /* SCOTCH_PTHREAD */
 
   o = stratParserParse (strattab, string);        /* Parse strategy string */
 
